### PR TITLE
Adapt the way queries are passed into Dune API

### PR DESCRIPTION
**Pull request by bh2smith**
Before we gave a file path, but soon we will want to manipulate and concatenate different SQL files. This allows that.

We also sneak in a change to the sample env file (that was missing relevant parameters).

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export DUNE_USER=
+export DUNE_PASSWORD=
+export DUNE_QUERY_ID=
+export FILE_OUT_PATH=./out
```
Comment (MartinquaXD) [inline .env.sample:4]: Unrelated to this PR.

File: src/dune_analytics.py [hunk 2]
```diff
@@ -342,24 +342,20 @@ def open_query(cls, filepath: str) -> str:
...
             self,
-            query_filepath: str,
+            query_str: str,
             network: Network,
             name: str,
             parameters: Optional[list[QueryParameter]],
     ) -> list[dict]:
         """
-        :param query_filepath: path to sql file to execute
+        :param query_str: path to sql file to execute
```
Comment (josojo) [inline src/dune_analytics.py:351]: Nit: Let's also adapt the comment

Comment (MartinquaXD) [inline src/dune_analytics.py:351]: > path to sql file to execute

Is no longer correct if you pass the query itself as a string.

**Review by josojo — APPROVED** [review]
THanks a lot!

File: .env.sample (hunk 1 above)
Comment (bh2smith) [reply .env.sample:4]: True but I'm leaving it here.

Comment (MartinquaXD) [reply .env.sample:4]: This is not a blocking issue for me, but I think those things should still be mentioned to foster a better PR etiquette in the long run.

Comment (bh2smith) [reply .env.sample:4]: Updated PR description.

File: src/dune_analytics.py (hunk 2 above)
Comment (bh2smith) [reply src/dune_analytics.py]: gefixed.

Comment (bh2smith) [reply src/dune_analytics.py]: gefixed.

**Pull request merged**